cinema/serializers.py

```diff
@@ -60,6 +70,8 @@ class MovieSessionListSerializer(MovieSessionSerializer):
         source="cinema_hall.capacity", read_only=True
     )
 
+    tickets_available = serializers.IntegerField(read_only=True)
```

Comment:
The `tickets_available` field is defined as read-only, but there is no logic provided to calculate or populate this field. Ensure that this field is correctly populated, possibly by overriding the `get_tickets_available` method or similar.